### PR TITLE
update findCsvFiles fn and add pkg.json scripts for parsing csv files

### PR DESCRIPTION
**Pull request by ebrahim-2**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. 
If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to static food data?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

PR purpose:
1- update findCsvFiles fn 
2- create a pkg.json script for every dataset to parse CSV files
3- fix the raw path problem

### CODE REVIEW

File: package/src/findCsvFiles.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ async function findCsvFiles(dirPath) {
 
   // eslint-disable-next-line no-shadow
   async function helper(dirPath) {
+    if (dirPath.includes('.json') || dirPath.includes('.md')) return;
```
Comment (atherdon) [inline package/src/findCsvFiles.js]: i don't like these short versions. make it casual

**Comment by atherdon** [comment]
another thing - maybe we dont need a separated file for our new method? because in the end it will be bundles to one file.

and tell me if you need help with calling scripts from pkg.json - not sure if you solve it or not

**Comment by atherdon** [comment]
will merge it and release a new version at npm

**Comment by atherdon** [comment]
new version released

File: package/src/findCsvFiles.js (hunk 1 above)
Comment (ebrahim-2) [reply package/src/findCsvFiles.js]: I will make it as regex pattern

**Comment by ebrahim-2** [comment]
> and tell me if you need help with calling scripts from pkg.json - not sure if you solve it or not

I didn't make it in that way I added the same script in every `package.json` file

**Comment by atherdon** [comment]
show me a code link

File: package/src/findCsvFiles.js (hunk 1 above)
Comment (atherdon) [reply package/src/findCsvFiles.js]: ok

**Comment by atherdon** [comment]
Btw, this is not what you're trying to do -  
https://github.com/GroceriStar/food-datasets-csv-parser/blob/master/package/src/fileSystem.js#L36? 

Looks pretty similar

**Comment by atherdon** [comment]
and same thing here too: https://github.com/GroceriStar/food-datasets-csv-parser/blob/master/package/src/csvToJson.js#L51

**Pull request merged**